.github/workflows/automatic-update.yml

```diff
@@ -43,3 +44,15 @@ jobs:
 
             **Notes for reviewer:**
             This is an auto-generated PR which ran `npx -y @grafana/create-plugin@latest update`, updates plugin.json files, grafana-plugin-sdk-go and docker-compose files for each plugin example
+
+      - name: Auto approve PR
+        uses: hmarr/auto-approve-action@v4
+        with:
+          pull-request-number: ${{ steps.create_pr.outputs.pull-request-number }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Enable auto-merge for PR
+        run: gh pr merge --auto --squash "$PR_URL"
```

Comment:
Where does `$PR_URL` come from?

Does it make sense to enable auto-merge for the PR before approving it?

Reply:
Ah ok I'm blind - the PR_URL is in the env below. 👍